spec/system/visit_sign_in_spec.rb

```diff
@@ -21,15 +21,15 @@
     sign_in('invalid@email.com', user.password)
 
     expect(current_path).to eql new_user_session_path
-    expect(page).to have_content 'Invalid Email or password.'
+    expect(page).to have_content 'Invalid Username or Password.'
   end
 
   scenario 'user cannot sign in with wrong password' do
     user = FactoryBot.create(:user)
-    sign_in(user.email, 'invalidpw')
+    sign_in(user.email, 'invalidpassword')
 
     expect(current_path).to eql new_user_session_path
-    expect(page).to have_content 'Invalid Email or password.'
+    expect(page).to have_content 'Invalid Username or Password.'
   end
```

Comment:
we should have the 2FA test and some controller test to check the different conditions...